.github/workflows/tests.yml

```diff
@@ -19,6 +19,8 @@ jobs:
             ${{ runner.os }}-node-
       - name: npm install
         run: "npm install"
+      - name: "Clean tree"
```

Comment:
actually, what is this checking? we're not currently building the artifact as part of CI (we should) so there's never going to be any difference

Reply:
That script includes the build

```
"test:clean-tree": "npm run build && sh scripts/check-for-changes.sh"
```